src/devices/video/upd7220.cpp

```diff
@@ -500,46 +481,46 @@ inline void upd7220_device::rdat(uint8_t type, uint8_t mod)
 //-------------------------------------------------
 //  write_vram -
 //-------------------------------------------------
-inline void upd7220_device::write_vram(uint8_t type, uint8_t mod, uint16_t data)
+inline void upd7220_device::write_vram(uint8_t type, uint8_t mod, uint16_t data, uint16_t mask = 0xffff)
```

Comment:
You shouldn’t have default argument values in more than one place.  They should really only be present at the declaration.  Having it at the definition here causes potential issues if they get out of sync.